### DISCUSSION TITLE
aws configure set default.s3.preferred_transfer_client crt  works well within Account

### DISCUSSION THREAD

**Discussion by esulliva51** [aws/aws-cli#8149] [Q&A]
I have had very good success copying within the same Account / Region for my development environment.  My production environment different account same region when copying the preferred_transfer_client seems to default back and not use CRT.   We are setting these two variables ; source_profile and Ec2InstanceMetadata in order to copy from an s3 bucket to an EC2 instance in a different account.  Is this a know limitation or is there another role that could be used or an account that has access to both without having to set the source profile and Ec2InstanceMeta columns.  Seems as thought it should work the same since within the same region.

**Comment by tim-finnigan** [comment]
Hi @esulliva51 thanks for reaching out. I'm not sure if I fully understand the issue or question here involving [preferred_transfer_client](https://awscli.amazonaws.com/v2/documentation/api/latest/topic/s3-config.html#preferred-transfer-client). Could you provide example commands/configurations to help with reproducing the issue? Also if you could provide logs (with any sensitive info redacted) by adding `boto3.set_stream_logger('')` to your script that may help with understanding the issue better.
